src/main/java/org/opensearch/security/authtoken/jwt/EncryptionDecryptionUtil.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.authtoken.jwt;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+
+public class EncryptionDecryptionUtil {
+
+    public static String encrypt(final String secret, final String data) {
+        final Cipher cipher = createCipherFromSecret(secret, CipherMode.ENCRYPT);
```

Comment:
If this method and `decrypt` will be used many times with the same secret then it may be better to instantiate a class and have cipher be created upon instantiation. This will create `cipher` every time that `encrypt` or `decrypt` are called.

Reply:
Good idea. I have refactor the class, and also made the changes for the file that used this.